tikv-configuration-file.md

```diff
@@ -402,17 +402,17 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 
 + The storage format and interface version used by TiKV when TiKV serves as the raw key-value store.
 + Value options:
-    + `1`: Uses API V1, does not encode the data passed to the client, and stores data as it is. In versions earlier than v6.1.0, TiKV uses API V1 by default.
+    + `1`: Uses API V1, does not encode the data passed from the client, and stores data as it is. In versions earlier than v6.1.0, TiKV uses API V1 by default.
     + `2`: Uses API V2:
-        + The data is stored in the MVCC (Multi Version Concurrency Control) format, where the timestamp (TSO) is obtained from PD by tikv-server.
-        + When API V2 is used, you are expected to set `storage.enable-ttl = true` at the same time. Because API V2 supports the TTL feature, it is mandatory to turn on `enable-ttl` in case that `enable-ttl` appears in contradiction with API V2.
+        + The data is stored in the MVCC (Multi Version Concurrency Control) format, where the timestamp is obtained from PD (for example, TSO) by tikv-server.
```

Comment:
```suggestion
        + The data is stored in the MVCC (Multi Version Concurrency Control) format, where the timestamp is obtained from PD (i.e., TSO) by tikv-server.
```

"timestamp obtained from PD" & `TSO` is the same thing.
See https://docs.pingcap.com/tidb/stable/glossary#tso.